src/main/charts/bamboo/values.yaml

```
@@ -784,6 +784,21 @@ monitoring:
   #   rules:
   #     - pattern: ".*"
 
+  serviceMonitor:
+
+    # -- Create ServiceMonitor to start scraping metrics. ServiceMonitor CRD needs to be created in advance.
+    #
+    create: false
+
+    # -- ServiceMonitorSelector of the prometheus instance.
+    #
+    prometheusLabelSelector: {}
+      # release: prometheus
+
+    # -- Scrape interval for the JMX service. Defaults to 30 seconds
+    #
+    scrapeIntervalSeconds:
```

```suggestion
    scrapeIntervalSeconds: 30
```

Then we can drop the second sentence in the comment.

---

Added defaults in values.yaml